impeller/entity/contents/contents.h

```diff
@@ -66,7 +66,10 @@ class Contents {
                       RenderPass& pass) const = 0;
 
   //----------------------------------------------------------------------------
-  /// @brief Get the screen space bounding rectangle that this contents affects.
+  /// @brief Get the pass space bounding rectangle that this contents affects.
```

Comment:
Perhaps something like "Get the area of the current render pass that is affected by these contents." The bit about the top-left relativity is simply whats in go/impeller-coordinates right? I suppose it doesn't hurt to reiterate.

Also, can we clarify what std::nullopt would mean and why we don't use empty rects?

Reply:
Ah, I meant to include that it's counted in pixels here. I updated this to a slightly more generic version "Get the area that will be affected when this contents is rendered" with a clarifying "During rendering, coverage coordinates count pixels from the top left corner of the framebuffer".

Added a note about the return values too. `std::nullopt` just means no coverage... I do think we should move to using empty coverage instead here in the future, since we also confusingly have a case where `std::nullopt` means "no bounds restriction" in Aiks (added much more recently, of course).